m4_sessions/1710-inning/group_four.md

```diff
@@ -1,31 +1,80 @@
-****Group Four****
-
-**Members**
+**Group Four Members**
 - David Becker
 - Jon Beckman
 - Sam Johnson
-- Yohanan Assefa
 - Michael Sagopolutele
 - Daniel Vogelsang
 
-Session Title: Our new role in social movements developers
-Description: Why should students come to this session?
+**Session Title: Social Movements And Technology**
+
+**Description:**
+
+In an increasingly technological world, there is an opportunity for those of us with the ability to write in the language of technology to play an oversized role in the direction of humanity. In this session, participants will be exposed to a number of social movements and the ways technology has played a role in them and then be given the opportunity to explore their own interests and ways they can make an impact. Participants will end the session with a tangible plan of action for their own personal involvement as developers after Turing.
+
+**Session Outcomes:** What will students leave with or gain from this session?
+
+A personal mission statmement/committment to the role they will play as a developer in making their communities a better place
+
+**Pre-Work/Communication**
+- Ask people to sign-up for [Code For Social Good](https://app.code4socialgood.org/) 
+- Ask for people to poke around on the site and pick out a project they are interested in
```

Comment:
Awesome that you are giving this out for people ahead of time.  Best way to do this is to create a slack channel so people can also share the projects they are interested in.

Reply:
@emhickmann We are planning to create a Slack channel for this!  When should we have our Pre-Work/Communication ready for this?

Reply:
@sljohnson32 I'll be giving you your group on Monday of week 5.  I usually have the groups ready to go by lunch.

Reply:
@emhickmann I've volunteered to create the slack channel and distribute the pre-work. Looking forward to next monday, but might need a reminder!